pom.xml

```diff
@@ -3,7 +3,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>org.testobject</groupId>
 	<artifactId>testobject-java-api</artifactId>
-	<version>0.2.1</version>
+	<version>0.2.0</version>
```

Comment:
version is not pushed to mvn repo yet. I thought **0.2.0** looks nicer and might work.